codeHintsInstall.php

```diff
@@ -0,0 +1,295 @@
+<?php
+/**
+ * Created by IntelliJ IDEA.
+ * User: Vladimir.Malinovskiy
+ * Date: 2/7/2017
+ * Time: 11:11 PM
+ */
+
+$docText =
+'/**
+ * Api client class
+ *
+ * The class is used to instantiate api client
+ *
+ * @package    Callfire
+ * @subpackage Api
+ * @author     Justin Martin <jmart@callfire.com>
+ */';
+
+$callFireSwaggerUrl = 'https://www.callfire.com/v2/api-docs/swagger.json';
+$spec = file_get_contents($callFireSwaggerUrl);
+$decodedSpec = json_decode($spec, true);
+$definitions = $decodedSpec['definitions'];
+$paths = $decodedSpec['paths'];
+foreach ($paths as $apiBunch) {
+    foreach ($apiBunch as $apiInstance) {
+        $docText .= "\n\n".loadDataToFile($apiInstance);
+    }
+}
+
+//UPDATING Client.php class to include documentation
+$file = './src/CallFire/Api/Client.php';
+$classCode = file_get_contents($file);
+$lastLineForPackagesDefinitions = 'use Swagger\Client as SwaggerClient;';
+$packageInfo = substr($classCode, 0, strpos($classCode, $lastLineForPackagesDefinitions) + strlen($lastLineForPackagesDefinitions));
+$classDefinition = substr($classCode, strpos($classCode, "class Client extends SwaggerClient"));
+$classCode = $packageInfo."\n\n".$docText."\n".$classDefinition;
+file_put_contents($file, $classCode);
```

Comment:
This will not be idempotent, I think. Each time the script is run, it seems like it will add more comments.